### PR TITLE
Homework nr 6

### PR DESCRIPTION
**Pull request by VictorTuricenco**

### CODE REVIEW

File: src/main/java/org/example/homework_nr_5/Weekday.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    MONDAY,
+    TUESDAY,
+    WEDNESDAY,
+    THURSDAY,
+    FRIDAY,
+    SATURDAY,
+    SUNDAY;
+
+        public boolean isWeekday() {
+            for (int index = 0; index != Weekday.values().length; index++)
```
Comment (gh3rtzz) [inline src/main/java/org/example/homework_nr_5/Weekday.java]: No need to loop, use `this` keyword instead
Also, no need for second return

File: src/main/java/org/example/homework_nr_5/Weekday.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+                return Weekday.values()[index] != SATURDAY && Weekday.values()[index] != SUNDAY;
+            return false;
+        }
+
+        public boolean isHoliday() {
+            for (int index = 0; index != Weekday.values().length; index++)
```
Comment (gh3rtzz) [inline src/main/java/org/example/homework_nr_5/Weekday.java]: Same here

File: src/main/java/org/example/homework_nr_5/Weekday.java (hunk 1 above)
Comment (VictorTuricenco) [reply src/main/java/org/example/homework_nr_5/Weekday.java]: It seems you've confused pull requests. We are talking about homework No 5 now (but the pull request is for homework6).
I did 2 commits in homework 5 initially - 1st version was done  using "this" and another one was with loops. I amended it again anyway.

**Review by gh3rtzz — APPROVED** [review]
approved

**Pull request merged**